src/sonic_ax_impl/mibs/ietf/rfc3433.py

```diff
@@ -424,6 +425,16 @@ def update_xcvr_dom_data(self):
 
             # skip RJ45 port
             transceiver_info_entry_data = Namespace.dbs_get_all(self.statedb, mibs.STATE_DB, mibs.transceiver_info_table(interface))
+    
+            if 'type' not in transceiver_info_entry_data:
+                # Only write error log once
+                if interface not in self.broken_transceiver_info:
+                    mibs.logger.error(
+                        "Invalid interface {} in STATE_DB, \
+                        attribute 'type' missing in transceiver_info '{}'".format(interface, transceiver_info_entry_data))
+                    self.broken_transceiver_info.append(interface)
```

Comment:
>broken_transceiver_info [](http://example.com/codeflow?start=25&length=23)

Not an issue and just comment: since the `broken_transceiver_info` is indexed by `interface`, and `interface` is validated above, so there are limited number of possible `interface`, so the `broken_transceiver_info` will have size limitation.

Reply:
Fixed by flush broken_transceiver_info in reinit_data()

Reply:
Sorry for misleading, my point is this is not an issue or a bug. Because `broken_transceiver_info` has limited size by its current usage. No need to reinit.

Reply:
Unnecessary change reverted.